client/src/Components/PlayerList.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+class PlayerList extends React.Component {
+
+    constructor() {
+        super();
+        this.state = {
+            players: []
+        }
+    }
+
+    componentDidMount() {
+        console.log("component did mount");
+        fetch('http://localhost:5000/api/players')
+            .then(res => res.json())
+            .then(result => { console.log(result); this.setState({ players: result }) })
+            .catch(err => console.log('Error fetching players.', err));
```

Comment:
Nice to have used the in-built fetch to get your data. What is the motivation for this?